feat(curriculum): add description list workshop to FSD cert (#62597)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-major-browsers-list/68ecfa1003d88990bb96a33d.md

---
id: 68ecfa1003d88990bb96a33d
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

Inside your `dl`, add a `dt` element containing the text `Google Chrome`, followed by a `dd` element dscribing it, containing the following text:

`This is a free web browser developed by Google and first released in 2008.`

# --hints--

Your page should contain a `dl` element.

```js
const dl = document.querySelector('dl');
assert.exists(dl);
```

Your `dl` element should contain at least one `dt` element.

```js
const dl = document.querySelector('dl');
const dt = dl?.querySelector('dt');
assert.exists(dt);
```

Your `dl` element should contain at least one `dd` element.

```js
const dl = document.querySelector('dl');
const dd = dl?.querySelector('dd');
assert.exists(dd);
```

> Your first `dt` element should mention `Google Chrome`.

Reviewer: ```suggestion
Your first `dt` element should have the text `Google Chrome`.
```

---
id: 68ecfa1003d88990bb96a33d
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

Inside your `dl`, add a `dt` element containing the text `Google Chrome`, followed by a `dd` element dscribing it, containing the following text:

`This is a free web browser developed by Google and first released in 2008.`

# --hints--

Your page should have a `h1` element.

```js
const h1 = document.querySelector('h1');
assert.exists(h1)
```

Your `h1` element should contain the text `List of Major Web Browsers`

```js
const h1 = document.querySelector('h1');
assert.include(h1.textContent, 'List of Major Web Browsers');
```

Your page should contain a `dl` element

```js
const dl = document.querySelector('dl')
assert.exists(dl)
```

Your `dl` element should contain at least one `dt` element

```js
const dl = document.querySelector('dl');
const dt = dl.querySelector('dt')
assert.exists(dt)
```

Your `dl` element should contain at least one `dd` element

```js
const dl = document.querySelector('dl');
const dd = dl.querySelector('dd')

> assert.exists(dd)

Reviewer 2: ```suggestion
const dl = document.querySelector('dl');
const dd = dl.querySelector('dd');
assert.exists(dd);
```

---
id: 68ecfa1003d88990bb96a33d
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--
Add your first term and description.

Inside your `<dl>`, add a `<dt>` element for **Google Chrome**, followed by a `<dd>` element dscribing it.

> > "This is a free web browser developed by Google and first released in 2008."

Reviewer 2: ```suggestion
`This is a free web browser developed by Google and first released in 2008.`
```

put text that needs to be added as is in backticks, please apply this to all the steps

---
id: 68ecfa1003d88990bb96a33d
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

Add your first term and description.

Inside your `dl`, add a `dt` element for **Google Chrome**, followed by a `dd` element dscribing it.

`This is a free web browser developed by Google and first released in 2008.`

# --hints--

Your page should have a `h1` element.

```js
const h1 = document.querySelector('h1');
assert.exists(h1)
```

Your `h1` element should contain the text "List of Major Web Browsers"

```js
const h1 = document.querySelector('h1');
assert.include(h1.textContent, 'List of Major Web Browsers');
```

Your page should contain a `dl` element

```js
const dl = document.querySelector('dl')
assert.exists(dl)
```

Your `dl` element should contain at least one `dt` element

```js
const dl = document.querySelector('dl');
const dt = dl.querySelector('dt')
assert.exists(dt)
```

Your `dl` element should contain at least one `dd` element

```js
const dl = document.querySelector('dl');
const dd = dl.querySelector('dd')
assert.exists(dd)
```

> Your first `dt` element should mention "Google Chrome".

Reviewer 2: ```suggestion
Your first `dt` element should mention `Google Chrome`.
```
